tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>&gt;Checkpoint and deserialize</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I deleted this section.</p>

Reply:
<p>Why did you delete it? </p><p> </p><p>You can checkpoint with Keras weights since weight clustering doesn't depend on the step.</p><p> </p><p>You still need <code>clustering_scope</code>  for Keras models. </p>

Reply:
<p>I thought it was not useful. OK, added it back.</p>